### PR TITLE
Add helper scripts needed for CI

### PR DESCRIPTION
**Pull request by fweikert**
- import_presubmit_config can be used to generate a YAML configuration file for a member project so that it can be tested as part of the future federation CI pipeline.
- create_project_workspace will be invoked by the CI to generate a WORKSPACE file for each project.

### CODE REVIEW

File: import_presubmit_config.py [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+import re
+import sys
+import urllib.request
+import utils
+import traceback
+import yaml
+
+# TODO(fweikert): keep this list in sync with PLATFORMS in https://github.com/bazelbuild/continuous-integration/blob/master/buildkite/bazelci.py
+# (or find a better solution)
+PYTHON_VERSIONS = {"debian10": "python3.7", "macos": "python3.7", "windows": "python.exe"}
```
Comment (aiuto) [inline import_presubmit_config.py:35]: nit: line wrap and indent 4.

File: import_presubmit_config.py [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+        return dumper.represent_scalar("tag:yaml.org,2002:str", data, style="|")
+
+    return dumper.represent_scalar("tag:yaml.org,2002:str", data)
+
+
+def get_project_name_from_url(config_url):
+    match = PROJECT_REGEX.search(config_url)
+    if not match:
+        raise Error(
+            "Config URL '%s' does not point to a file in the bazelbuild GitHub organisation."
```
Comment (aiuto) [inline import_presubmit_config.py]: nit, line wrap.

File: utils.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+
+
+def eprint(msg):
+    """
```
Comment (aiuto) [inline utils.py]: """ commants should have a single sentance on the first line, so

""" Print to stderr and flush (just in case)."""

**Review by aiuto — CHANGES_REQUESTED** [review]

File: import_presubmit_config.py (hunk 1 above)
Comment (fweikert) [reply import_presubmit_config.py:35]: The formatter doesn't like that change :(

**Pull request merged**